FOUNDATIONS.md

```diff
@@ -39,46 +39,81 @@ the format. Therefore, we should avoid introducing new formats unnecessarily as
 
 ## No Non-Local Reasoning
 
-An IPLD block should never be interpreted in the context of *anything* not
-contained in the block (and CID).
+Transforming content of a Block into Nodes conforming to the IPLD
```

Comment:
I'd like to see clarification that the original authors of the text see the same thing here. The whole discussion about "context" makes me think that there could be some disagreement boiling beneath this one. I like the way you've made it more specific but it seems that it might be more specific than others may want.

Reply:
+1 to would enjoy more eyes on this.  We've **got** to do something about the "anything" and broadness of the original, though, in some way or other.  "interpreted" is also a trigger word for me: data can be interpreted multiple ways simultaneously, so that's also thus a very touchy word to use here -- some sort of "interpreted for {purpose}" is necessary, imo.

Reply:
If we can add _more_ "doing X must not require interpretation with any more information than contained in Y+Z" entries, I'd see that as a very welcome improvement.

Reply:
I guess I can see how the way I've rephased this can be seen as being a blacklist, whereas "always" in computer science the better answer is to whitelist.  Unfortunately, I don't see any choice: due to the "data can be interpreted multiple ways" fact, I don't see any other way to go about this which doesn't result in trying to outlaw the sun shining.  (How shall I interpret an S-expr serialized in IPLD?  In the context of a Lisp Machine, of course.  But also as the text: homoiconicity!  *thumps SICP unironically for once*)